### ISSUE TITLE
Guake 0.5.0-3 crashes on opening

### ISSUE THREAD

**Issue by tomecki** [Guake/guake#440]
Guake 0.5.0-3 installed from Fedora 21 package crashes every time I try to open it. 
Here is the exceprt from crash report:

python2.7 killed by SIGFPE
kernel version: 3.17.4-301.fc21.x86_64
```
{   "signal": 8
,   "executable": "/usr/bin/python2.7"
,   "stacktrace":
      [ {   "crash_thread": true
        ,   "frames":
              [ {   "address": 140509100689167
                ,   "build_id": "7d2c61871b22ac1471c0ceef44b259efecc27bdf"
                ,   "build_id_offset": 231183
                ,   "function_name": "_vte_draw_set_text_font"
                ,   "file_name": "/lib64/libvte.so.9"
                }
              , {   "address": 140509100596035
                ,   "build_id": "7d2c61871b22ac1471c0ceef44b259efecc27bdf"
                ,   "build_id_offset": 138051
                ,   "function_name": "vte_terminal_ensure_font"
                ,   "file_name": "/lib64/libvte.so.9"
                }
              , {   "address": 140509100596471
                ,   "build_id": "7d2c61871b22ac1471c0ceef44b259efecc27bdf"
                ,   "build_id_offset": 138487
                ,   "function_name": "vte_terminal_size_request"
                ,   "file_name": "/lib64/libvte.so.9"
                }
              , {   "address": 140509310725395
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 77075
                ,   "function_name": "g_cclosure_marshal_VOID__BOXEDv"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310713522
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 65202
                ,   "function_name": "_g_closure_invoke_va"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310819168
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 170848
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310822650
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 174330
                ,   "function_name": "g_signal_emit_by_name"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509266328090
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1833498
                ,   "function_name": "_gtk_size_group_compute_requisition"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509265041229
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 546637
                ,   "function_name": "gtk_box_size_request"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509310725395
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 77075
                ,   "function_name": "g_cclosure_marshal_VOID__BOXEDv"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310713522
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 65202
                ,   "function_name": "_g_closure_invoke_va"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310819168
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 170848
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310822650
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 174330
                ,   "function_name": "g_signal_emit_by_name"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509266328090
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1833498
                ,   "function_name": "_gtk_size_group_compute_requisition"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509265979651
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1485059
                ,   "function_name": "gtk_notebook_size_request"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509310725395
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 77075
                ,   "function_name": "g_cclosure_marshal_VOID__BOXEDv"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310713522
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 65202
                ,   "function_name": "_g_closure_invoke_va"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310819168
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 170848
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310822650
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 174330
                ,   "function_name": "g_signal_emit_by_name"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509266328090
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1833498
                ,   "function_name": "_gtk_size_group_compute_requisition"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509265041229
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 546637
                ,   "function_name": "gtk_box_size_request"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509310725395
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 77075
                ,   "function_name": "g_cclosure_marshal_VOID__BOXEDv"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310713522
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 65202
                ,   "function_name": "_g_closure_invoke_va"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310819168
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 170848
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310822650
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 174330
                ,   "function_name": "g_signal_emit_by_name"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509266328090
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1833498
                ,   "function_name": "_gtk_size_group_compute_requisition"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509267150485
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 2655893
                ,   "function_name": "gtk_window_size_request"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509310725395
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 77075
                ,   "function_name": "g_cclosure_marshal_VOID__BOXEDv"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310713700
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 65380
                ,   "function_name": "_g_closure_invoke_va"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310819168
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 170848
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310822650
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 174330
                ,   "function_name": "g_signal_emit_by_name"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509266328090
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1833498
                ,   "function_name": "_gtk_size_group_compute_requisition"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509267145981
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 2651389
                ,   "function_name": "gtk_window_compute_configure_request"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509267153381
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 2658789
                ,   "function_name": "gtk_window_show"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509310713141
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 64821
                ,   "function_name": "g_closure_invoke"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310784332
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 136012
                ,   "function_name": "signal_emit_unlocked_R"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310820737
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 172417
                ,   "function_name": "g_signal_emit_valist"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509310821295
                ,   "build_id": "cdb5e508362f0ebe4bf74e3414e7d83caee7fb66"
                ,   "build_id_offset": 172975
                ,   "function_name": "g_signal_emit"
                ,   "file_name": "/lib64/libgobject-2.0.so.0"
                }
              , {   "address": 140509267105126
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 2610534
                ,   "function_name": "gtk_widget_show"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509272604861
                ,   "build_id": "3f9c297aa57da8825f589bc89e33cb43d41d7679"
                ,   "build_id_offset": 1245373
                ,   "function_name": "_wrap_gtk_widget_show_all"
                ,   "file_name": "/usr/lib64/python2.7/site-packages/gtk-2.0/gtk/_gtk.so"
                }
              , {   "address": 140509470644252
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 927772
                ,   "function_name": "PyEval_EvalFrameEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470647296
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 930816
                ,   "function_name": "PyEval_EvalCodeEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470640868
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 924388
                ,   "function_name": "PyEval_EvalFrameEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470647296
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 930816
                ,   "function_name": "PyEval_EvalCodeEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470172636
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 456156
                ,   "function_name": "function_call"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470021891
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 305411
                ,   "function_name": "PyObject_Call"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470083093
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 366613
                ,   "function_name": "instancemethod_call"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470021891
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 305411
                ,   "function_name": "PyObject_Call"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470618823
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 902343
                ,   "function_name": "PyEval_CallObjectWithKeywords"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509065927756
                ,   "build_id": "b4972a09e1cb54c40d6dbd26e78c999019064eea"
                ,   "build_id_offset": 15436
                ,   "function_name": "caller"
                ,   "file_name": "/usr/lib64/python2.7/site-packages/guake/globalhotkeys.so"
                }
              , {   "address": 140509065925652
                ,   "build_id": "b4972a09e1cb54c40d6dbd26e78c999019064eea"
                ,   "build_id_offset": 13332
                ,   "function_name": "filter_func"
                ,   "file_name": "/usr/lib64/python2.7/site-packages/guake/globalhotkeys.so"
                }
              , {   "address": 140509261987395
                ,   "build_id": "749a837b77ed0251943b2187d481141893427528"
                ,   "build_id_offset": 384579
                ,   "function_name": "gdk_event_translate"
                ,   "file_name": "/lib64/libgdk-x11-2.0.so.0"
                }
              , {   "address": 140509261995750
                ,   "build_id": "749a837b77ed0251943b2187d481141893427528"
                ,   "build_id_offset": 392934
                ,   "function_name": "_gdk_events_queue"
                ,   "file_name": "/lib64/libgdk-x11-2.0.so.0"
                }
              , {   "address": 140509261995918
                ,   "build_id": "749a837b77ed0251943b2187d481141893427528"
                ,   "build_id_offset": 393102
                ,   "function_name": "gdk_event_dispatch"
                ,   "file_name": "/lib64/libgdk-x11-2.0.so.0"
                }
              , {   "address": 140509317753579
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 301803
                ,   "function_name": "g_main_context_dispatch"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317754504
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 302728
                ,   "function_name": "g_main_context_iterate.isra.29"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317755314
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 303538
                ,   "function_name": "g_main_loop_run"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509265825447
                ,   "build_id": "a7d0ce8ab7f201fda25df9a3aee8ad74c608eac2"
                ,   "build_id_offset": 1330855
                ,   "function_name": "gtk_main"
                ,   "file_name": "/lib64/libgtk-x11-2.0.so.0"
                }
              , {   "address": 140509273121815
                ,   "build_id": "3f9c297aa57da8825f589bc89e33cb43d41d7679"
                ,   "build_id_offset": 1762327
                ,   "function_name": "_wrap_gtk_main"
                ,   "file_name": "/usr/lib64/python2.7/site-packages/gtk-2.0/gtk/_gtk.so"
                }
              , {   "address": 140509470644252
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 927772
                ,   "function_name": "PyEval_EvalFrameEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470647296
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 930816
                ,   "function_name": "PyEval_EvalCodeEx"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470647545
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 931065
                ,   "function_name": "PyEval_EvalCode"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470750991
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 1034511
                ,   "function_name": "run_mod"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470755634
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 1039154
                ,   "function_name": "PyRun_FileExFlags"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470760263
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 1043783
                ,   "function_name": "PyRun_SimpleFileExFlags"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                }
              , {   "address": 140509470831338
                ,   "build_id": "505486efa1764d1d523b29e746583663cbda5fe6"
                ,   "build_id_offset": 1114858
                ,   "function_name": "Py_Main"
                ,   "file_name": "/lib64/libpython2.7.so.1.0"
                } ]
        }
      , {   "frames":
              [ {   "address": 140509457187293
                ,   "build_id": "b9debd8d4a1d247dcbcb4b6675f1567c113fcf0e"
                ,   "build_id_offset": 1003997
                ,   "function_name": "poll"
                ,   "file_name": "/lib64/libc.so.6"
                }
              , {   "address": 140509317754404
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 302628
                ,   "function_name": "g_main_context_iterate.isra.29"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317755314
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 303538
                ,   "function_name": "g_main_loop_run"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509307911574
                ,   "build_id": "c0eba3baa4a34e0172cd966e8cc167a3e25a534f"
                ,   "build_id_offset": 896406
                ,   "function_name": "gdbus_shared_thread_func"
                ,   "file_name": "/lib64/libgio-2.0.so.0"
                }
              , {   "address": 140509317912501
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 460725
                ,   "function_name": "g_thread_proxy"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509467534634
                ,   "build_id": "1bf31aa5658439b345305715922edf83126ce4bc"
                ,   "build_id_offset": 29994
                ,   "function_name": "start_thread"
                ,   "file_name": "/lib64/libpthread.so.0"
                }
              , {   "address": 140509457233789
                ,   "build_id": "b9debd8d4a1d247dcbcb4b6675f1567c113fcf0e"
                ,   "build_id_offset": 1050493
                ,   "function_name": "__clone"
                ,   "file_name": "/lib64/libc.so.6"
                } ]
        }
      , {   "frames":
              [ {   "address": 140509457187293
                ,   "build_id": "b9debd8d4a1d247dcbcb4b6675f1567c113fcf0e"
                ,   "build_id_offset": 1003997
                ,   "function_name": "poll"
                ,   "file_name": "/lib64/libc.so.6"
                }
              , {   "address": 140509317754404
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 302628
                ,   "function_name": "g_main_context_iterate.isra.29"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317754684
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 302908
                ,   "function_name": "g_main_context_iteration"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317754745
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 302969
                ,   "function_name": "glib_worker_main"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509317912501
                ,   "build_id": "4aa20c4aca9bb539c0dfdfa176f765c9d4b388a8"
                ,   "build_id_offset": 460725
                ,   "function_name": "g_thread_proxy"
                ,   "file_name": "/lib64/libglib-2.0.so.0"
                }
              , {   "address": 140509467534634
                ,   "build_id": "1bf31aa5658439b345305715922edf83126ce4bc"
                ,   "build_id_offset": 29994
                ,   "function_name": "start_thread"
                ,   "file_name": "/lib64/libpthread.so.0"
                }
              , {   "address": 140509457233789
                ,   "build_id": "b9debd8d4a1d247dcbcb4b6675f1567c113fcf0e"
                ,   "build_id_offset": 1050493
                ,   "function_name": "__clone"
                ,   "file_name": "/lib64/libc.so.6"
                } ]
        } ]
}
```

**Comment by pypingou** [comment]
Start guake-prefs and set it to use the system fonts

**Comment by tomecki** [comment]
Funny thing, I had system fonts enabled, but doing exactly the opposite: switching to custom font did the thing.  Thanks :)

But, if someone wants default fonts, this is still a bug.

**Comment by Stibbons** [comment]
Should have been fixed in 0.5.1
